src/coreclr/jit/compiler.cpp

```diff
@@ -934,6 +939,7 @@ var_types Compiler::getArgTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
     }
 
     return useType;
+#endif // TARGET_WASM
```

Comment:
```suggestion
#endif // !TARGET_WASM
```
The convention for the last `#ifdef`s is to signify what is above it.